app/services/foreman/access_permissions.rb

```diff
@@ -210,24 +210,32 @@
 
   permission_set.security_block :external_variables do |map|
     map.permission :view_external_variables,    {:lookup_keys => [:index, :show, :auto_complete_search],
+                                                 :puppetclass_lookup_keys => [:index, :show, :auto_complete_search],
+                                                 :variable_lookup_keys => [:index, :show, :auto_complete_search],
                                                  :lookup_values => [:index],
                                                  :"api/v1/lookup_keys" => [:index, :show],
                                                  :"api/v2/smart_variables" => [:index, :show],
                                                  :"api/v2/smart_class_parameters" => [:index, :show],
                                                  :"api/v2/override_values" => [:index, :show]}
     map.permission :create_external_variables,  {:lookup_keys => [:new, :create],
```

Comment:
or in another question, where do we use the lookup_keys permission, vs the new two ones?